README.md

```diff
@@ -2,13 +2,18 @@
 
 Run tests with statements and method calls removed to help identify broken tests
 
-Install from [crates.io]:
+## Installation
+
+#### System requirements:
+Install `pkg-config` and `libsqlite3-dev` on your system.
```

Comment:
Could we somehow mention that these are the Debian/Ubuntu package names? (I think RedHat packages have different names, for example, but we certainly don't have to list them all.)

Also, could you please run `prettier` on this file? I think `prettier` is complaining because it wants a blank line after the heading.

Reply:
I went ahead and did this. I hope you don't mind. 😬

Reply:
thanks :+1: